utilities/tuminer/README.md

```diff
@@ -62,8 +62,8 @@ optional arguments:
 nvidia-docker run -it \
   -v $PWD/test/corpus:/root/corpus:/corpus \
   -v /tmp/output:/output \
-  opennmt/tuminer \
-  --mode score \
+  nmtwizard/tuminer \
```

Comment:
Should it be `nmtwizard/tuminer tuminer` instead?

Reply:
Ok this is fixed.

Reply:
just updated